elf2flt.c

```diff
@@ -1892,9 +1895,9 @@ int main(int argc, char *argv[])
 
     if (sec_vma < *vma) {
       if (*len > 0)
-	*len += sec_vma - *vma;
+        *len += sec_vma - *vma;
```

Comment:
let's leave the whitespace alone in here.  this file is already a mismash of tabs-vs-spaces, so no sense in tweaking this one location.

Reply:
Okay, no worries.  Whitespace change removed in new force-pushed commit.